website/docs/search.md

```diff
@@ -37,22 +37,19 @@ By default, the Docusaurus preset generates a [sitemap.xml](https://docusaurus.i
 
 :::
 
-### Index Configuration {#algolia-index-configuration}
+:::info From an old docsearch version?
 
-After applying, your site's DocSearch config should be created at:
+You can read more about migration from DocSearch v2 and configurating v3 in [our blog post](/blog/2021/11/21/algolia-docsearch-migration).
```

Comment:
Actually v2/v3 is the UI, we don't have versioning for the infra, it's just `legacy` for the previous one.

We can remove the versions since Docusaurus v2 only support DocSearch v3 UI components